concordium-consensus/src/Concordium/Queries.hs

```diff
@@ -575,34 +575,38 @@ getTransactionStatusInBlock trHash blockHash =
 checkIsCurrentFinalizer :: MVR gsconf finconf Bool
 checkIsCurrentFinalizer = liftSkovQueryLatest isFinalizationCommitteeMember
 
+-- |Check whether consensus has been shut down
+checkIsShutDown :: MVR gsconf finconf Bool
+checkIsShutDown = liftSkovQueryLatest isShutDown
+
 -- |Result of a baker status query.
 data BakerStatus
     = -- |The baker is a member of the current committee
-      ActiveBaker !BakerId
+      ActiveBaker
     | -- |The account has a baker, but it is not yet in the committee
-      InactiveBaker !BakerId
+      InactiveBaker
     | -- |The baker id does not correspond with a current baker
       NoBaker
     | -- |The baker may exist, but the keys do not match
-      BadKeys !BakerId
+      BadKeys
```

Comment:
Can we match the names between the Haskell and Rust versions?

Reply:
Yes, that would make sense.